en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 
 ## New features
 
-#### API Governance
+??? note "JDK 17 support"
+    You can now run our products with JDK 17
 
-??? note "Management of real-time APIs on Solace"
+??? note "OpenTelemetry Support"
+    OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends
 
-    WSO2 API Manager 4.1.0 now supports integrating external third-party Gateways and Brokers. Solace is an advanced event broker that is efficiently streaming events and information across cloud, on-premises, and IoT environments.
+??? note "ELK Support for MI"
+    From WSO2 API Manager 4.2.0 onwards, Micro Integrator now enables publishing operational analytics for Elastic stack as a replacement for WSO2 EI Analytics.
 
-    **[Learn more]({{base_path}}/tutorials/integrating-with-solace/)**
+??? note "Configuring Self Sign up through Admin Portal"
+    WSO2 API Manager now offers you the ability to self-signup into the Developer Portal as well as the option to disable self-signup. You can also migrate the self-signup configurations from the registry through the **Advance Configuration** option or through editing the `tenant-conf.json` file.
 
-??? note "Develop an integration from a managed API"
+??? note "Configuring API Lifecycle through Admin Portal"
+    APIs created in WSO2 API Manager have their own life cycle consisting of the following: a set of life cycle states, specific actions for each state transition, and a checklist of items before a state transition occurs. An API has a predefined life cycle consisting of six states by default. This feature allows you to customize and/or add new states, migrate API Lifecycle through the **Advance Configuration** option or through editing the `tenant-conf.json` file.
 
-    With the API Manager 4.1.0 release, you can create an integration using a managed API and configure the integration service endpoint to the managed API. This is the top-down approach in API first integration.
+??? note "Enable logs without server restart"
+    In this new version of WSO2 API Manager, you can enable or disable correlation logs as needed for troubleshooting at runtime, as a server restart is not required.
 
-    **[Learn more]({{base_path}}/tutorials/develop-an-integration-with-a-managed-api/)**
+??? note "Removal of Jaggery Apps from Portals"
+    Jaggery is not currently in an active stage of development or upkeep. Therefore, it is essential to switch to a different language which is closer to server-side while still using the react component as the client-side handler.
```

Comment:
now JSP is used as the server side language for all portals. All jaggeryapps have been migrated to webapps.